src/plots/gl3d/scene.js

```diff
@@ -714,6 +714,7 @@ proto.setConvert = function() {
     for(var i = 0; i < 3; ++i) {
         var ax = this.fullSceneLayout[axisProperties[i]];
         Axes.setConvert(ax, this.fullLayout);
+        ax.setScale = Lib.noop;
```

Comment:
This statement also appears in the 3d [`axis_defaults`](https://github.com/plotly/plotly.js/blob/9f4823be4f50239503bb3c5da126db3980619384/src/plots/gl3d/layout/axis_defaults.js#L56).